### PR TITLE
Clean up packaging scripts

### PR DESCRIPTION
**Pull request by penev92**
Cleans up the packaging scripts and updates them to better match what we have in the OpenAL-CS/SDL2-CS/FreeType repos.

https://github.com/penev92/Eluant/actions/runs/4876852889
https://www.nuget.org/packages/OpenRA-Eluant/1.0.21-a6

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+; Unix-style newlines
+[*]
+end_of_line = LF
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+; 4-column tab indentation
+[*.yml]
+indent_style = space
```
Comment (abcdefg30) [inline .editorconfig:12]: Space?

Comment (penev92) [reply .editorconfig:12]: GitHub Actions workflows normally use spaces for indentation. We can probably change that if you want.

Comment (abcdefg30) [reply .editorconfig:12]: Not strictly necessary (although I prefer tabs). We should be consistent with the comment above though.

Comment (penev92) [reply .editorconfig:12]: image
So I'll just remove the misleading comment from the `.editorconfig`.

**Comment by penev92** [comment]
Removed the `.editorconfig` comment.

**Review by abcdefg30 — APPROVED** [review]

**Comment by penev92** [comment]
This PR would be as good a place as any to add the proposed CI build from https://github.com/OpenRA/SDL2-CS/pull/7, but IMO we should have it run the `build-wrapper` and `build-natives` workflows rather than `build-nuget`, because that one will try to package and publish a NuGet package to nuget.org on every open PR and every push to a PR.

**Comment by penev92** [comment]
> Added workflow run for PRs as a CI check

**Pull request merged**